spec/lib/id_p_settings_adapter_spec.rb

```diff
@@ -3,7 +3,9 @@
 RSpec.describe IdPSettingsAdapter do
   describe 'IdPSettingsAdapter.mock_saml?' do
     it 'return false' do
-      expect(IdPSettingsAdapter.mock_saml?).to eq false
+      mock_saml_setting = Rails.configuration.x.laa_portal[:mock_saml]
```

Comment:
I think `laa_portal.mock_saml` works instead of `laa_portal[:mock_saml]`

Reply:
That's better - thank you - though implementing your other suggestion removes this line.